pep-0001.txt

```diff
@@ -281,14 +281,24 @@ notifying the PEP author(s) and giving them a chance to make revisions.
 The final authority for PEP approval is the Steering Council. However, whenever
 a new PEP is put forward, any core developer that believes they are suitably
 experienced to make the final decision on that PEP may offer to serve as
-the PEP-Delegate for that PEP, and they will then have the authority to approve
-(or reject) that PEP. Individuals taking on this responsibility are free to seek
+the PEP-Delegate (BDFL-Delegate) for that PEP, and they will then have the
+authority to approve (or reject) that PEP. 
+
+The terms, PEP-Delegate (preferred term under the Steering Council governance 
+model) and BDFL-Delegate, may be used interchangeably in discussion.
+The term "PEP-Delegate" will be used under the Steering
+Council governance model. The PEP's designated decision maker,
+the "PEP-Delegate" with the Steering Council's support, is
+recorded in the "BDFL-Delegate" field in the PEP's header. For consistency with
+PEPs written prior to the Steering Council's governance model, the field name,
+"BDFL-Delegate" will still be kept but the field means "PEP-Delegate".
```

Comment:
We actually ended up changing this, and new PEPs use the PEP-Delegate field:
```
pep-0639.rst:PEP-Delegate: Paul Moore <p.f.moore at gmail.com>
pep-0641.rst:PEP-Delegate: Pablo Galindo <pablogsal@python.org>
pep-0656.rst:PEP-Delegate: Paul Moore <p.f.moore@gmail.com>
pep-0658.rst:PEP-Delegate: Donald Stufft <donald@stufft.io>
pep-0665.rst:PEP-Delegate: Paul Moore <p.f.moore@gmail.com>
```

Reply:
Should we note that the PEP-Delegate field is used with the advent of the SC model, but BDFL-Delegate is a legacy alias? (Just ensuring we explain what both mean, as I think it is unlikely we will mass update all old PEPs to use PEP-Delegate.)

Reply:
Good suggestion. Nope, we don't want to update the old PEPs.

Reply:
> Should we note that the PEP-Delegate field is used with the advent of the SC model, but BDFL-Delegate is a legacy alias? (Just ensuring we explain what both mean, as I think it is unlikely we will mass update all old PEPs to use PEP-Delegate.)

:+1: That's also the approach taken in other governance-related PEPs that mention this.